CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.10.2
+
+- Upgrade minimum SDK to `^3.10.0` and the databases to [2025c].
```

Comment:
Thanks for the update! Please note the new version in the [README](https://github.com/srawlins/timezone/blob/master/README.md) as well. I think just in the second paragraph.